Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.7-slim
+
+RUN apt update && apt-get install -y libpq-dev gcc
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5942194)